vkcs/internal/acctest/acctest.go

```diff
@@ -195,3 +197,14 @@ func ImportStep(resourceName string) resource.TestStep {
 		ImportStateVerify: true,
 	}
 }
+
+func GenerateTags(testName string) map[string]string {
```

Comment:
`GenerateTags` is a slightly misleading function name in the context of having cloud objects tags when you use it for names as well.

Reply:
change